src/utils/table.ts

```diff
@@ -1,4 +1,4 @@
-import { DEFAULT_TABLE_ROWS, DEFAULT_TABLE_COLUMNS } from "settings";
```

Comment:
I dont think we use `DEFAULT_TABLE_ROWS` anymore. If this is the case, delete it from the settings, please

Reply:
`DEFAULT_TABLE_ROWS` is used in calculation of `DEFAULT_TABLE_CAPACITY` in `src/settings.ts`
and in `src/components/molecules/TablesUserList/TablesUserList.tsx`

Reply:
Okay, sounds good